scripts/git.py

```diff
@@ -1,10 +1,8 @@
-from subprocess import (
-    PIPE,
-    run,
-)
+from subprocess import run
+from typing import Sequence
 
 
-def tags():
+def tags() -> Sequence[str]:
```

Comment:
Does mypy explicit requests Sequence or would be List also ok, because if requests Sequence, we have a small hell, before us

Reply:
It does not require a Sequence, but a list is also a sequence ([see here](https://docs.python.org/3/library/collections.abc.html#collections-abstract-base-classes-1)). I usually try to use the interface with the least constrains
in order to have more freedom to change the used type in the future. Honestly ideally it should return a `iterator` if the user needs a `sequence` or `list` the  call site should call `list(tags())`.  Due to the fact that there was already one user/call site expecting to get at least a sequence type (index operator), I made it a sequence for now.

After writing this down I think we should change `def tags()` to return an `Iterable` :D as mentioned above, what do you think?

Reply:
Hm, good question. If a generator is possible, we can do this, but then we need to careful at the caller sites. On the other hand, you never know how many tags you get, so the generator might the be saver.